generator/js/cards.js

```diff
@@ -321,7 +322,11 @@ function card_generate_color_style(color, options) {
 }
 
 function card_generate_color_gradient_style(color, options) {
-    return 'style="background: radial-gradient(ellipse at center, white 20%, ' + color + ' 120%)"';
+    if(options.icon_gradient){
```

Comment:
Code formatting is not entirely as the rest of the project. But it's fine as it'll probably get a big refactor anyway!